### PR TITLE
Update DomF to dunfell release

### PR DESCRIPTION
**Pull request by oleksiimoisieiev**

### CODE REVIEW

**Review by al1img — COMMENTED** [review]
Why do we need ostree for DomF?

File: recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-extended/netconfig/files/eth.network [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[Match]
```
Comment (al1img) [inline recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-extended/netconfig/files/eth.network]: How did it work before without netconfig?

File: recipes-domu/domu-image-fusion/domu-image-fusion.bbappend [hunk 2]
```diff
@@ -18,7 +18,7 @@ XT_QUIRK_BB_ADD_LAYER_append = " \
 # Generic ARMv8
 ################################################################################
 SRC_URI += " \
-    repo://github.com/xen-troops/manifests;protocol=https;branch=master;manifest=prod_aos/domf.xml;scmdata=keep \
+    repo://github.com/oleksiimoisieiev/manifests;protocol=https;branch=domf_dunfell;manifest=prod_aos/domf.xml;scmdata=keep \
```
Comment (al1img) [inline recipes-domu/domu-image-fusion/domu-image-fusion.bbappend]: Should be removed?

File: recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-aos/aos-iamanager/aos-iamanager_git.bbappend [hunk 3]
```diff
@@ -28,6 +28,11 @@ FILES_${PN} += " \
     /var/aos/finish.sh \
 "
 
+do_compile_prepend(){
```
Comment (al1img) [inline recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-aos/aos-iamanager/aos-iamanager_git.bbappend:32]: Is it still required after moving to go 1.14?

Comment (oleksiimoisieiev) [reply recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-aos/aos-iamanager/aos-iamanager_git.bbappend:32]: Requires dunfell release on domd yocto

File: recipes-domu/domu-image-fusion/domu-image-fusion.bbappend (hunk 2 above)
Comment (oleksiimoisieiev) [reply recipes-domu/domu-image-fusion/domu-image-fusion.bbappend]: removed

File: recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-extended/netconfig/files/eth.network (hunk 1 above)
Comment (oleksiimoisieiev) [reply recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-extended/netconfig/files/eth.network]: removed

**Pull request merged**